doc/guides/west/index.rst

```diff
@@ -4,30 +4,29 @@ West (Zephyr's meta-tool)
 #########################
 
 The Zephyr project includes a swiss-army knife command line tool named
-``west``\ [#west-name]_. West is developed in its own `repository`_.  Like
-``git`` and ``docker``, the top-level ``west`` command takes some common
+``west``\ [#west-name]_. West is developed in its own `repository`_.
+
+West's built-in commands provide a multiple repository management system with
+features inspired by Google's Repo tool and Git submodules. West is also
+"pluggable": you can write your own west extension commands which add
+additional features to west. Zephyr uses this to provide conveniences for
+building applications, flashing and debugging them, and more.
+
+Like ``git`` and ``docker``, the top-level ``west`` command takes some common
 options, a sub-command to run, and then options and arguments for that
 sub-command::
 
   west [common-opts] <command> [opts] <args>
 
-West's built-in commands provide a multiple repository management
-system with features inspired by Google's Repo tool and Git
-submodules. West simplifies configuration and is also pluggable: you
-can write your own west "extension commands" which add additional
-features to west.  Zephyr uses this feature to provide conveniences
-for building applications, flashing and debugging them, and more.
+Since west v0.8, you can also run west like this::
 
-It is possible not to use west for Zephyr development if you do not
-require these features, prefer to use your own tools, or want to
-eliminate the extra layer of indirection. However, this implies extra
-effort and expert knowledge.
```

Comment:
Do we really want to take this section out?

Reply:
> Do we really want to take this section out?

I think the index has gotten kind of long with it still there now that there's additional text related to running west added. There's still a link to "using zephyr without west" in the toctree so it's not gone from the page, just less emphasized. I'm happy to put it back if you think it's important.

Reply:
Ah, right. No need to keep this, then.